policy/modules/admin/tomoyo-auditd.fc

```diff
@@ -0,0 +1,5 @@
+/etc/tomoyo/tools/auditd\.conf	--	gen_context(system_u:object_r:tomoyo_auditd_etc_t,s0)
```

Comment:
use "conf" instead of "etc". etc refers to /etc and conf is more flexible (also i would consider targeting all of /etc/tomoyo)
lets say you have a "tomoyo-auditd confined admin" , now you have to give him access to rw etc_t dirs to that he can create symlinks in /etc/tomoyo//tools.

Reply:
ok